src/app/parcels/form/clientDatabaseFunctions.ts

```diff
@@ -3,11 +3,14 @@ import { InsertSchema, UpdateSchema } from "@/databaseUtils";
 import { logErrorReturnLogId, logWarningReturnLogId } from "@/logger/logger";
 import { AuditLog, sendAuditLog } from "@/server/auditLog";
 
-type ParcelDatabaseInsertRecord = InsertSchema["parcels"];
-type ParcelDatabaseUpdateRecord = UpdateSchema["parcels"];
+export type ParcelDatabaseInsertRecord = InsertSchema["parcels"];
+export type ParcelDatabaseUpdateRecord = UpdateSchema["parcels"];
 
-type InsertParcelErrors = "failedToInsertParcel";
-type InsertParcelReturnType = { error: { type: InsertParcelErrors; logId: string } | null };
+export type InsertParcelErrors = "failedToInsertParcel";
+export type InsertParcelReturnType = {
+    error: { type: InsertParcelErrors; logId: string } | null;
+    parcelId: string | undefined;
```

Comment:
this is such a nitpick but please change the order of these lines, and in the function, change the return to be `return {parcelId: ..., error: ...}` instead of `return {error: ..., parcelId: ...}`

Similarly for updateParcel.

And also please change the order in line 233 in ParcelForm.tsx

Reply:
Yep, happy do do that, may I ask why? I guess the parcelId is the more important field than the error? Does it match consistency with something?

Reply:
yep, it's just that in the rest of the codebase we return the data and then the error, i suppose the data is more important in some way? but its the consistency that i think we care about